edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -305,6 +326,27 @@ def _is_jwt_cookie_and_session_user_mismatch(self, request):
 
         return True
 
+    def _is_jwt_and_lms_user_email_mismatch(self, request, user):
+        """
+        Returns True if user email in JWT and email of user do not match, False otherwise.
+
+        Arguments:
+            request: The request.
+            user: user from user_and_auth
+        """
+        lms_user_email = getattr(user, 'email', None)
+
+        # This function will check for token in the authorization header and return it
+        # otherwise it will return token from JWT cookies.
+        token = JSONWebTokenAuthentication.get_token_from_request(request)
+        decoded_jwt = configured_jwt_decode_handler(token)
+        jwt_user_email = decoded_jwt.get('email', None)
+
+        if lms_user_email != jwt_user_email:
```

Comment:
Nit: You could just return the result, but this is up to you.
```suggestion
        return lms_user_email != jwt_user_email:
```